### DISCUSSION TITLE
Issue when deleting a database function

### DISCUSSION THREAD

**Discussion by JustinBerghahn** [supabase/supabase#28152] [Questions] [answered]
Heya, i'm trying to delete a database function. But when i try to delete it i get `Failed to delete database function: cannot drop function create_profile_for_new_user() because other objects depend on it` back. I don't have any triggers or a table, so i was wondering if somebody can help me with is.

<img width="397" alt="Screenshot 2024-07-23 at 9 52 14 PM" src="https://github.com/user-attachments/assets/124ada50-785a-4862-ad0c-53f03237d9a4">

**Comment by GaryAustin1** [comment, marked as answer]
You likely do have a trigger in auth schema for that function.
`DROP FUNCTION name CASCADE;`
From the SQL editor should do it, or you need to drop the trigger first from the SQL editor.

**Reply by JustinBerghahn** [reply]
Thanks a lot!
